### DISCUSSION TITLE
Improve Refiner Functionality

### DISCUSSION THREAD

**Discussion by softtaco1** [Panchovix/stable-diffusion-webui-reForge#218] [Ideas]
I would love to see more control over the refiner being added! These would be things such as:

-Ability to swap sampler after certain percentage of generation.
-Ability to swap scheduler after certain percentage of generation.
-Ability to swap CFG Scale after certain percentage of generation.
-etc

Thank you for your continued support of this passion project, @Panchovix!

**Comment by tripped144** [comment]
I second this. I've been searching for an extension and I don't think there is one. It would be super nice to be built into the refiner. Some models I use the refiner with need a very different cfg than the original model, so it just doesn't work.
